raspbian/applications/README.md

```diff
@@ -4,3 +4,7 @@
     - Information on the demonstration camera software
 - [OMXPlayer](omxplayer.md)
     - Information on the OMXPlayer command line media player
+- [Minecraft for pi]
```

Comment:
@JamesH65 Perhaps this is supposed to be linking to `usage/minecraft/README.md` ?

Reply:
Have added link and removed the electronic circuit bit, which is a feature, but don't need to reference it here I don't think.